test/test.pem

```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
@durran please confirm that the contents of this file are safe to add to a public repository :)